### PR TITLE
1220 turn off parsers

### PR DESCRIPTION
**Pull request by millmanorama**
Added a ingest settings panel for the plaso module with check boxes for the pe and winreg parsers (off by default).  Turned off all firefox, chrome, and IE related parsers.

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    .addComponent(peCheckBox)
+                    .addComponent(winRegCheckBox))
+                .addContainerGap())
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addContainerGap()
+                .addComponent(jTextArea1, javax.swing.GroupLayout.DEFAULT_SIZE, 117, Short.MAX_VALUE)
+                .addGap(18, 18, 18)
```
Comment (kellykelly3) [inline Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java]: Is there a reason to use a hard coded gap instead of a preferred gap?

**Comment by rcordovano** [comment]
@millmanorama, please address merge conflicts.

@kellykelly3 please indicate whether or not you approve this PR.

File: Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+        org.openide.awt.Mnemonics.setLocalizedText(peCheckBox, org.openide.util.NbBundle.getMessage(PlasoModuleSettingsPanel.class, "PlasoModuleSettingsPanel.peCheckBox.text")); // NOI18N
+        peCheckBox.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                peCheckBoxActionPerformed(evt);
+            }
+        });
+
+        jTextArea1.setEditable(false);
+        jTextArea1.setBackground(new java.awt.Color(240, 240, 240));
```
[the remark this answers is not in the document]
Comment (millmanorama) [reply Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java]: used UIDefaults

File: Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+     * form. WARNING: Do NOT modify this code. The content of this method is
+     * always regenerated by the Form Editor.
+     */
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        winRegCheckBox = new javax.swing.JCheckBox();
+        peCheckBox = new javax.swing.JCheckBox();
+        jTextArea1 = new javax.swing.JTextArea();
```
[the remark this answers is not in the document]
Comment (millmanorama) [reply Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java]: done

File: Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java (hunk 1 above)
Comment (millmanorama) [reply Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoModuleSettingsPanel.java]: nope. fixed

**Comment by rcordovano** [comment]
@millmanorama, we have merge conflicts again. I made an attempt to resolve them, but lost confidence as I went along.

I did notice that the Plaso ingest module has code to pop up message boxes in the process method. Please take a few minutes to remove this code and make it log / return with an error as appropriate - I imagine the original author did not know that ingest modules need to be able to run unattended.

**Pull request merged**